src/plugin.rs

```diff
@@ -80,40 +88,40 @@ pub fn load(config: &ClientConfig) -> Result<Vec<Box<dyn Plugin>>> {
     }
 
     // For all the plugins we know, try to enable them.
-    let mut ret: Vec<Box<dyn Plugin>> = Vec::new();
+    let mut ret: Vec<_> = Vec::new();
```

Comment:
Why do you need the type at all here? Can this just be:
```rust
let mut ret = Vec::new();
```

Reply:
Good catch